training/templates/training_template.tpl.php

```diff
@@ -0,0 +1,3 @@
+<!--<div></div>-->
```

Comment:
remove this entire block as it doesn't make sense

<!--<div></div>-->
<?php //drupal_render($variables['']) ?>
<?php //var_dump($variables['']); ?>